dll/win32/browseui/shellbrowser.cpp

```diff
@@ -350,6 +350,7 @@ class CShellBrowser :
     HRESULT UpdateUpState();
     void UpdateGotoMenu(HMENU theMenu);
     void UpdateViewMenu(HMENU theMenu);
+    HRESULT ShowSysMenuContextMenu(INT xPos, INT yPos);
```

Comment:
Or "ShowSysContextMenu" would be shorter.

Reply:
Renamed.